src/components/Avatar/index.js

```diff
@@ -6,14 +6,18 @@ const Avatar = (props) => {
     width: ${props.size || '80'}px;
     border-radius: ${props.Circle ? '50%' : '0'};
     background-color: #9B9B9B;
```

Comment:
This color isnt available on our collor pallete?

Reply:
Good point, I checked and it's not available, but I added a grey color that is available in our color pallete in Theme. It's pretty much the same and I think there's no need to add this especific color into our color pallete. I think we are good to go with the regular grey that I picked from Theme.